torch/csrc/jit/pickle.h

```diff
@@ -69,7 +69,8 @@ TORCH_API IValue pickle_load(const std::vector<char>& data);
 TORCH_API IValue unpickle(
```

Comment:
Can we add some comments to the pickle and unpickle APIs in terms of what the type_resolve is?